src/compiler/utilities.ts

```diff
@@ -1884,8 +1884,8 @@ namespace ts {
             writeTextOfNode,
             writeLiteral,
             writeLine,
-            increaseIndent: () => indent++,
-            decreaseIndent: () => indent--,
+            increaseIndent: () => { indent++; },

```

Comment:
This is a meaningful change - these functions no longer return the indent level. Does this represent a shortcoming in the rule, and should they be reverted?